torchci/pages/flakytest.tsx

```diff
@@ -50,6 +52,7 @@ export default function Page() {
                   })}
                 </ul>
               </div>
+              {samples?.length > 0 && <div><p>Example logs: </p> <LogViewer job={samples[0]} /></div>}
```

Comment:
This just gets the first sample--maybe in the future we can put one for every job